lamindb/models/_from_values.py

```diff
@@ -22,56 +22,55 @@ def get_or_create_records(
     field: StrField,
     *,
     create: bool = False,
-    from_source: bool = False,
     organism: Record | str | None = None,
     source: Record | None = None,
     mute: bool = False,
+    _from_source: bool = False,
```

Comment:
Why is this private, now?

Could you document all arguments, please? Or provide a comment that explains that all of these parameters are present in `from_values()`?

Could we rename this function to `from_values()`?